openstudiocore/src/model/GeneratorPhotovoltaic.cpp

```diff
@@ -136,7 +136,7 @@ namespace detail {
     return "Generator:Photovoltaic";
   }
 
-  boost::optional<double> GeneratorPhotovoltaic_Impl::ratedThermalToElectricalPowerRatio() const
+  boost::optional<double> GeneratorPhotovoltaic_Impl::ratedThermaltoElectricalPowerRatio() const
```

Comment:
@jmarrec meh, I'm not crazy about either way here, I guess I prefer the original casing